cmsdb/processes/ewk.py

```diff
@@ -50,130 +52,141 @@
     xsecs={13: Number(0.1)},  # TODO
 )
 
+# NNLO cross section, based on:
+# https://twiki.cern.ch/twiki/bin/viewauth/CMS/StandardModelCrossSectionsat13TeV
+
 dy_lep_m50 = dy_lep.add_process(
     name="dy_lep_m50",
     id=51100,
-    xsecs={13: Number(0.1)},  # TODO
+    xsecs={13: const.n_leps * Number(6077.22, {
+        "integration": 1.49,
+        "scale": 0.02j,
+        "pdf": 14.78,
+    })},
 )
 
+# based on datasets DY{i}JetsToLL_M-50_MatchEWPDG20_TuneCP5_13TeV-madgraphMLM-pythia8 (Summer20UL16, LO)
 dy_lep_m50_1j = dy_lep_m50.add_process(
     name="dy_lep_m50_1j",
     id=51111,
-    xsecs={13: Number(0.1)},  # TODO
+    xsecs={13: Number(928.3)},  # TODO
```

Comment:
Does the TODO still apply? If not, this could trigger unwanted search results when looking for TODO tags :)